src/coursework/entitys/Employee.java

```diff
@@ -0,0 +1,92 @@
+package coursework.entitys;
+
+import coursework.enums.Department;
+
+import java.util.Objects;
+
+public class Employee {
+    private static long idCount;
+    private final long id;
+    private String name;
+    private String surName;
+    private String patronymic;
+    private Department department;
+    private double salary;
+
+    public Employee(String name, String surName, String patronymic, Department department, double salary) {
+        this.name = name;
+        this.surName = surName;
+        this.patronymic = patronymic;
+        this.department = department;
+        this.salary = salary;
+        idCount++;
+        this.id = idCount;
```

Comment:
Можно вот так написать:
```suggestion
        this.id = ++idCount;
```